<?xml version="1.0"?>
<launch>
	<arg name="data_path" default="$(find pose_cnn)/../../data/" />
	<arg name="models_dir" default="$(find pose_cnn)/../../data/models/" />

	<include file="$(find pose_cnn)/launch/recognition_server.launch"/>

	<node pkg="pose_cnn" type="posecnn_refined" name="posecnn_refined" args="$(arg data_path) $(arg models_dir)" output="screen" respawn="true"/>
</launch>